src/main/java/Company.java

```diff
@@ -0,0 +1,102 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Company {
+    private static final int MAX_EMPLOYEES = 100;
+    Employee[] employees = new Employee[MAX_EMPLOYEES];
```

Comment:
wszędzie gdzie to możliwe oznaczaj pola jako private

Reply:
Przestawiłem pole Employee[] jako pole metody readEmployees oraz poprawiłem metodę tak, żeby zwracała tablicę pracowników.